### PR TITLE
WIP : Some cosmetic improvements

### PR DESCRIPTION
**Pull request by jmMeessen**
I propose the following improvements:

- "sell" => "sales_pitch"
- resized the images/jenkins-cd-orchestrator.jpg to correct the proportions
- added the command to check the installed version (vagrant & virtualbox) 
- clean vs clear
- solve vs resolve
- improve "unit-test", "package" and "integration test" slides
- improve the simple docker command slides

### CODE REVIEW

File: chapters/04-demoapp.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ cat pom.xml
 
 == Demo Application: DevBox Tricks
 
-* Clear the window: `clean`
+* Clean the window: `clear`
```
Comment (dduportal) [inline chapters/04-demoapp.adoc:67]: ```-bash: clean: command not found``` ;)

+1 for replacing the 1st word, but the shell command is `clear`

File: chapters/03-hello.adoc [hunk 2]
```diff
@@ -17,27 +17,28 @@ image::{imagedir}/damien.jpg[height="400",float="left"]
...
-</sell>
+</sales_pitch>
 
 == Who are you ?
 
 == Requirements
 
-* VirtualBox >= 5.1.18
-* Vagrant >= 1.9.1
+* VirtualBox >= 5.1.18 &nbsp; &nbsp; &nbsp; ("_vboxmanage --version_")
```
Comment (dduportal) [inline chapters/03-hello.adoc]: `&nbsp;` should not be used on the source. Asciidoctor provides the `+` char to provide "forced" line returns (unless I misunderstood).
=> I'm not very pleased by pushing too much design in the content. What do you think about adding a sub list item with the command ?
=> Can you make the command to be a source block or inline code ?

File: chapters/03-hello.adoc [hunk 3]
```diff
@@ -17,27 +17,28 @@ image::{imagedir}/damien.jpg[height="400",float="left"]
...
+* Vagrant >= 1.9.1  &nbsp; &nbsp; &nbsp; ("_vagrant version_")
```
Comment (dduportal) [inline chapters/03-hello.adoc]: Same here: no `&nbsp;` and source code

File: chapters/04-demoapp.adoc [hunk 4]
```diff
@@ -108,8 +108,8 @@ ls -l ./target
 == Demo Application: Unit-Test it
 
 * Maven goal is *`test`*
-** Solves dependencies for unit test
-** Call `compile`
+** Execute `compile` goal
+** Resolve unit test dependencies
```
Comment (dduportal) [inline chapters/04-demoapp.adoc]: I totally agree on the rephrasing. However, the order matters: can you just invert those two rephrased sentences ?

File: chapters/04-demoapp.adoc [hunk 5]
```diff
@@ -124,10 +124,10 @@ ls -l ./target/surefire-reports
 == Demo Application: Build it
 
 * Maven goal is *`package`*
-** Solves dependencies for packaging
-** Call `compile` and `test`
+** Execute `compile` and `test` goals
+** Resolve packaging dependencies
```
Comment (dduportal) [inline chapters/04-demoapp.adoc]: Same here: invert :)

File: chapters/04-demoapp.adoc [hunk 6]
```diff
@@ -159,11 +159,11 @@ image::./images/test-integration.gif[height=300,float=right]
 == Demo Application: Integration Testing
 
 * Maven goal is *`verify`*
-** Solves dependencies for integration tests
-** Call `compile`, `test` and `package`
+** Execute `compile`, `test` and `package` goals
+** Resolve integration test dependencies
```
Comment (dduportal) [inline chapters/04-demoapp.adoc:161]: Same here, invert :)

**Review by dduportal — CHANGES_REQUESTED** [review]

File: chapters/04-demoapp.adoc (hunk 4 above)
Comment (jmMeessen) [reply chapters/04-demoapp.adoc]: removed the resolve line

File: chapters/04-demoapp.adoc (hunk 5 above)
Comment (jmMeessen) [reply chapters/04-demoapp.adoc]: removed the resolve line

File: chapters/04-demoapp.adoc (hunk 6 above)
Comment (jmMeessen) [reply chapters/04-demoapp.adoc:161]: removed the resolve line

File: chapters/03-hello.adoc (hunk 3 above)
Comment (jmMeessen) [reply chapters/03-hello.adoc]: Changed the presentation

File: chapters/03-hello.adoc [hunk 7]
```diff
@@ -17,27 +17,40 @@ image::{imagedir}/damien.jpg[height="400",float="left"]
...
 ** Jenkins "Enterprise" Distribution
 ** Services around Jenkins
 
-</sell>
+</sales_pitch>
 
 == Who are you ?
 
-== Requirements
+== prepare lab environment
```
Comment (dduportal) [inline chapters/03-hello.adoc:32]: After thinking about this 2 days, I really don't want to put those commands in a slide: it is "too much" distraction.

I might prefer to put links to vagrant and virtualbox website, and let people search a bit.

**Review by dduportal — APPROVED** [review]

**Pull request merged**